alacritty_terminal/src/term/cell.rs

```diff
@@ -131,7 +130,7 @@ impl Cell {
     #[inline]
     pub fn reset(&mut self, template: &Cell) {
         // memcpy template to self.
```

Comment:
I feel like this comment should be updated or removed.

Reply:
No worries, chances are the non-copy cells PR will do just that.